go.mod

```diff
@@ -66,8 +66,8 @@ require (
 	github.com/opentracing/opentracing-go v1.2.0
 	github.com/ory/dockertest/v3 v3.8.1
 	github.com/percona/mongodb_exporter v0.31.2
-	github.com/prometheus-community/elasticsearch_exporter v1.2.1
-	github.com/prometheus-community/postgres_exporter v0.10.0
+	github.com/prometheus-community/elasticsearch_exporter v1.5.0
+	github.com/prometheus-community/postgres_exporter v0.11.1
```

Comment:
These are user-facing changes so we should update the CHANGELOG to indicate a version bump. Ditto for statsd_exporter.